ui/src/router/components/configuration/components/experiment_config_section/variables/VariablesConfigTable.scss

```diff
@@ -1,13 +1,5 @@
 .euiPanel--configSection.variables {
   .euiDescriptionList {
     margin-left: 10px;
-
-    .euiDescriptionList__title {
-      width: 95px;
-    }
-
-    .euiDescriptionList__description {
-      width: calc(100% - 95px);
-    }
   }
```

Comment:
I removed these intentionally because they didn't quite look similar to the other config values displayed without the same column:

![Screenshot 2024-07-15 at 17 46 18](https://github.com/user-attachments/assets/07ca68f9-a334-4021-91e5-fa50d247a938)